pkgs/os-specific/linux/kernel/linux-testing-bcachefs.nix

```diff
@@ -1,24 +1,34 @@
-{ lib, buildPackages, fetchFromGitHub, fetchpatch, perl, buildLinux, ... } @ args:
+{ lib
+, fetchpatch
+, kernel
+, date ? "2021-07-03"
+, commit ? "56c86f90b122a3f1844359e8bead7227b5488863"
+, diffHash ? "1ilsxrmxnxs7wn4gj44bbqr4as8a97ivpm0mdbn5iqv3lbjjr30l"
```

Comment:
Are those the default for the none unstable variant? That is weird. They should be moved to that entry and explicitly set.

Reply:
I don't understand what you mean by that?
There is no version of bcachefs that is not unstable. These are specified as arguments to ease overriding, as running a newer, older, or alternative version of bcachefs is a common need at the current state of development.

Reply:
> There is no version of bcachefs that is not unstable.

Then we don't need this. The same can be achieve with overrideAttr.

Reply:
`overrideAttr` requires complete reconstruction of the content of the `kernelPatches` attribute, which is the opposite of 'easy'.

Reply:
Oh, well then we do it another time.